### ISSUE TITLE
Physical vs mental "poisoned" condition

### ISSUE THREAD

**Issue by itamarcu** [mlenser/kryx-rpg-issues#736]
Mechanically, "Poisoned" is simple disadvantage but limited to attack rolls and skill checks.

It would be interesting and flavorful if different kinds of poisons, venoms, and diseases inflicted different debilitating effects - affecting not only attacks and ability checks but also saving throws and spellcasting, and split in different ways.

My suggestion is to have several kinds of effects like "poisoned" (which aren't necessarily related to poison!).
1. **Clumsy / Feeble / Weakened** - You have disadvantage on attack rolls and ability checks that use Strength, Dexterity, or Constitution.
2. **Frail / Sickly** - You have disadvantage on Reflex and Fortitude saving throws, and <creatures have advantage on attack rolls against you / you have disadvantage on Defense rolls / your Defense is reduced by 4> (any of these will work)
3. **Lethargic / Stupefied** - You have disadvantage on attack rolls and ability checks that use Intelligence, Wisdom, or Charisma, and on Will saving throws. (maybe also advantage for creatures that make saving throws against spells you cast while having this condition)

For existing monsters and spells and items that make a creature "poisoned", I would suggest picking one of these three - depending on lore and flavor - and then saying something like "...on a failure, the creature is poisoned, making a saving throw against the poison at the end of each of its turns / every minute.  While poisoned, the creature is clumsy/frail/lethargic.".

I suggested three poisons here - offensive physical, defensive physical, and mental - because I feel like those three would be roughly even in how much they debilitate a character.  You could split to offensive+defensive, or mental+physical, or 6+ options for different abilities - but I wouldn't want to have too much complexity, and wouldn't want the condition to be as bad as e.g. Stunned.

**Comment by mlenser** [comment]
Conditions added need to have current uses or have options that can be changed to them.

- Clumsy / Feeble / Weakened: I don't see any usage of these. Ray of Enfeeblement causes half damage with weapon attacks.
- Frail / Sickly: I don't see any usage of these.
- Lethargic / Stupefied: I don't see any usage of these. Feeblemind is very different.

> For existing monsters and spells and items that make a creature "poisoned", I would suggest picking one of these three - depending on lore and flavor - and then saying something like "...on a failure, the creature is poisoned, making a saving throw against the poison at the end of each of its turns / every minute. While poisoned, the creature is clumsy/frail/lethargic.".

Someone (not me) would need to go through the 66+ monsters and all the poison spells and asign each to one of these.

Though I think it's good to first look over PF2e conditions to see if we can take some ideas from there for solidifying these conditions.

**Comment by mlenser** [comment]
FYI PF2e has the following conditions:
- [Clumsy](https://2e.aonprd.com/Conditions.aspx?ID=3)
- [Drained](https://2e.aonprd.com/Conditions.aspx?ID=10)
- [Enfeebled](https://2e.aonprd.com/Conditions.aspx?ID=13)
- [Stupefied](https://2e.aonprd.com/Conditions.aspx?ID=37)

Seems like they split it to Dex (Clumsy), Con (Drained), Str (Enfeebled), and mental (Stupefied).

Seems like a good path to follow imo.

**Comment by Marcloure** [comment]
I don't understand what is the purpose of this split, as I don't think there is a poison that makes you weaker without also making you slower, and poisons that target the nervous system usually also makes you worse at reacting with your body. When a snake or spider bites you, a lot of different things inside you stop to work, the same is true when you ingest cyanide.

What is the goal here? What kind of poison would apply only some of those conditions?

**Comment by itamarcu** [comment]
Yup, that's been my inspiration for the suggestion, but I thought it would be slightly more balanced to have 2 physical conditions (offense vs defense) instead of 3 physical conditions (str, dex, con).

Either way, I'm in favor of making a change like this.  I'll try to find time to go through all the poisoned-applying spells/concoctions, to see if they could be split to these 3-4 types.

@mlenser Could you give a text list of all monsters that cause the poisoned condition?

**Comment by Marcloure** [comment]
Dividing that is... weird. You can't do boxing that well with a headache, mental poisons also hinders your capacity to hit hard. It is also hard to take a math test when your arm is painfully heavy from a tetanus vaccine. In D&D, I can't think of a poison that would apply only some of those conditions.

>  it hurts martials a lot more than casters

This is true, but because casters don't rely on rolling to deal damage, it's their target that rolls. Creating a condition that only affects mental stats won't change that. Either their spells/maneuvers DC would have to decrease, or they would have to gain a spellcasting failure chance, neither of which needs a new condition. It can just be a new bullet point under poisoned.

**Comment by mlenser** [comment]
Irregardless of poisons: I think it makes sense to add enfeebled as a condition as we have Ray of Enfeeblement. I'm sure I could find some others that would do the same thing.

**Comment by mlenser** [comment]
> I don't understand what is the purpose of this split

There are plenty of different types of poisons. It could be good to categorize them. See PF1e for some examples: https://www.d20pfsrd.com/gamemastering/afflictions/poison

I'd have to check pf2e for theirs.

**Comment by mlenser** [comment]
I'm curious where this could go, but this is a lot of work. If you drive this idea (please suggest some ideas along the way to not waste time), then I can imlement it, but I don't have time to drive this myself.
